ranges/bitranges.nim

```diff
@@ -123,6 +130,12 @@ iterator enumerateBits(x: BitRange): (int, bool) =
     inc p
     inc i
 
+proc getBit*(bytes: openarray[byte], pos: Natural): bool =
+  getAbsoluteBit(bytes, pos)
+
+proc setBitBE*(bytes: var openarray[byte], pos: Natural, value: bool) =
```

Comment:
ugh, why is `BE` in the name here and not everywhere else?

Reply:
ah never mind, just saw that the code is littered with these endianess concerns.. looks a bit out of place - a byte array does not really have an endianness, unless it's bit-endianess which for sure would be unusual